frontend/nginx-custom.conf

```diff
@@ -1,8 +1,19 @@
 server {
-  listen 80;
-  location / {
-    root /usr/share/nginx/html;
-    index index.html index.htm;
-    try_files $uri $uri/ /index.html =404;
-  }
+    listen 80;
+
+    location / {
+        root /usr/share/nginx/html;
+        index index.html index.htm;
+        try_files $uri $uri/ /index.html =404;
+    }
+
+    location /api {
+        proxy_pass http://h2ms:8080/api;
```

Comment:
I think that will need to be `h2ms.org`

https://docs.nginx.com/nginx/admin-guide/web-server/reverse-proxy/

Reply:
It's not, this is the hostname the backend docker container has. It's internal communication in the docker network

Reply:
(It comes from https://github.com/2018swecapstone/h2ms/blob/master/docker-compose.yml#L21)

Reply:
Ahh, thanks for clearing that up. Approved.